.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js: node
-script: npm test
```

Comment:
Should this have been removed?

Reply:
The build fails when this line is present.